src/test/java/htsjdk/samtools/util/GZIIndexTest.java

```diff
@@ -72,16 +72,18 @@ public void testWriteIndex(final File indexFile, final int exprectedBlocks) thro
         return new Object[][] {
                 // TODO - requires more test files
```

Comment:
this seems like a good set of tests files since the two cases of 1 block and multiple blocks are both tested

Reply:
This is out of date, because at the beginning I haven't add the FASTA file. I removed the comment.